diptest/phase_test.go

```diff
@@ -126,8 +128,14 @@ func withStartedGameOptsAndOrders(conf func(m map[string]interface{}), orders or
 	startedGameEnvs = envs
 	startedGameDesc = gameDesc
 
+	executed := false
 	for phaseOrdinalMinus1, set := range orders {
 		set.execute(phaseOrdinalMinus1 + 1)
+		fmt.Print("p")
+		executed = true
+	}
+	if executed {
+		fmt.Println()
 	}
```

Comment:
...and this is even less informative ;)

Reply:
Also just progress indication thing - we want a new line after all the pooooooppooooop :)

Reply:
Poop! Haw haw. XD

Reply:
:D